services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api.py

```diff
@@ -29,6 +31,53 @@
 log = logging.getLogger(__name__)
 
 
+def _monkey_patch_aiodocker() -> None:
+    """Raises an error once the library is up to date."""
+    from distutils.version import LooseVersion
+
+    from aiodocker import volumes
+    from aiodocker.utils import clean_filters
+    from aiodocker.volumes import DockerVolume
+
+    if LooseVersion(aiodocker.__version__) > LooseVersion("0.21.0"):
+        raise RuntimeError(
+            "Please check that PR https://github.com/aio-libs/aiodocker/pull/623 "
+            "is not part of the current bump version. "
+            "Otherwise, if the current PR is part of this new release "
+            "remove monkey_patch."
+        )
+
+    # pylint: disable=protected-access
+    # pylint: disable=redefined-builtin
+    async def list(self, *, filters=None):
```

Comment:
rename it as ``_list`` or equivalently...  Redefining build-ins (as the linter is telling you) tends to be really error prone

```python
volumes.DockerVolumes.list = _custom_volumes_list
...
```

Reply:
Ahh yes, this one is awkward. The "list" is used in aiodocker's API. Will not change as the idea was to extend aiodocker's API. I've also submitted a PR but no one is taking care of it 😞